src/main/kotlin/com/mutualmobile/praxisspringboot/services/orgs/UserProjectService.kt

```diff
@@ -0,0 +1,12 @@
+package com.mutualmobile.praxisspringboot.services.orgs
+
+import com.mutualmobile.praxisspringboot.data.ApiResponse
+import com.mutualmobile.praxisspringboot.data.user.HarvestUserProject
+import org.springframework.http.ResponseEntity
+
+interface UserProjectService {
+    fun assignProjectToUser(
+        projectId: String,
+        userId: String
+    ): ResponseEntity<ApiResponse<HarvestUserProject>>
```

Comment:
@shubhamsinghmutualmobile the service should not return `ResponseEntity` it can return `HarvestUserProject`